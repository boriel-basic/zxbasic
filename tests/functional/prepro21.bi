' Must return 
' 10

#define macro(x) x(10) 
#define test(y) y 

macro(test)

