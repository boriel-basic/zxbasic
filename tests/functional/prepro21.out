#line 1 "prepro21.bi"






10  

